Google Play Music/Google Play Music.csproj

```diff
@@ -320,6 +320,9 @@
   </Target>
   <Import Project="..\packages\cef.redist.x86.3.2357.1287\build\cef.redist.x86.targets" Condition="Exists('..\packages\cef.redist.x86.3.2357.1287\build\cef.redist.x86.targets')" />
   <Import Project="..\packages\CefSharp.Common.43.0.0\build\CefSharp.Common.targets" Condition="Exists('..\packages\CefSharp.Common.43.0.0\build\CefSharp.Common.targets')" />
+  <PropertyGroup>
+    <PreBuildEvent>taskkill /F /IM CefSharp.BrowserSubprocess.exe 2&gt;&amp;1 | exit /B 0</PreBuildEvent>
```

Comment:
This is a compilation issue caused by the process locking assemblies and migration to Program.cs will not correct the error. I'll be more clear in my next commit messages.

Additionally, glancing through the issues list didn't show similar problems with installed instances. If they are reported, the process killing can be added to Program.cs. I'm concerned for other applications that may be using the same process which will be terminated unintentionally, whereas the coding machine is in a more expected state than the installed machine.